client/src/components/BookingRequest/BookingRequest.tsx

```diff
@@ -0,0 +1,115 @@
+import { Typography, Card, Box, TextField, Button } from '@material-ui/core';
+import { Rating } from '@material-ui/lab';
+import useStyles from './useStyles';
+import { Profile } from '../../interface/Profile';
+
+import { createBookingRequest } from '../../helpers/APICalls/requestBooking';
+import {Formik, FormikHelpers} from 'formik';
+import * as Yup from 'yup';
+import {useSnackBar} from '../../context/useSnackbarContext';
+import {useAuth} from '../../context/useAuthContext';
+
+interface Props {
+  profile: Profile;
+}
+
+interface FormProps {
+    startDate: string;
+    endDate: string;
+}
+
+
+export default function BookingRequest({ profile }: any): JSX.Element {
+  const classes = useStyles();
+  const {loggedInUser} = useAuth(); 
+  const {updateSnackBarMessage} = useSnackBar();
+  console.log(profile)
+
+    const handleSubmit = (values: FormProps, {setSubmitting}: FormikHelpers<FormProps>): void => {
+        const {startDate, endDate} = values;
+        const userId = loggedInUser?.id ? loggedInUser?.id : "";
+        const sitterId = profile?.userId
+
+    createBookingRequest({startDate, endDate, userId, sitterId}).then((data) => {
+            if(data.error) {
+            setSubmitting(false);
+            updateSnackBarMessage(data.error);
+            } else if (data.success) {
+            setSubmitting(false);
+            updateSnackBarMessage(data.success);
+            }
+        });
+    };
+
+
+  return (
+    <Card className={classes.bookingCard}>

```

Comment:
Looks good! Maybe you can add a max width to the card to preserve aspect ratio :)